Screens/ScheduleScreen.js

```diff
@@ -8,23 +8,24 @@ import { BallIndicator } from 'react-native-indicators';
 
 import PerformanceList from 'TheaterSchedule/Screens/ScheduleScreenComponents/PerformanceList'
 import DateFilter from 'TheaterSchedule/Screens/ScheduleScreenComponents/DateFilter';
-import { filterPerformances } from 'TheaterSchedule/Actions/ScheduleActions/ScheduleActionCreators'
+import { loadPerformances } from 'TheaterSchedule/Actions/ScheduleActions/ScheduleActionCreators'
 
 class ScheduleScreen extends Component {
     static navigationOptions = {
         drawerIcon: <MaterialCommunityIcons name='calendar-clock' size={25} />
     }
 
-    componentWillMount() {
-        const DAYS_IN_WEEK = 7;
+    componentDidUpdate(prevProps) {
+        if (prevProps.languageCode !== this.props.languageCode) {
```

Comment:
i think it will be better
  if (this.props.languageCode && !prevProps.languageCode ) {
--- load
    }